src/sidebar/util/oauth-client.js

```diff
@@ -246,15 +241,7 @@ export default class OAuthClient {
 
     // Generate settings for `window.open` in the required comma-separated
     // key=value format.
-    const authWindowSettings = queryString
-      .stringify({
-        left,
-        top,
-        width,
-        height,
-      })
-      .replace(/&/g, ',');
-
+    const authWindowSettings = `left=${left},top=${top},width=${width},height=${height}`;
```

Comment:
I presume this string formatting is more straightforward than using any `URLSearchParams` support? It seems it.

Reply:
Right. The values here are all just numbers (digits, dots) so no escaping is required.